### PR TITLE
Deleting many commands will be now much faster

### PR DESCRIPTION
**Pull request by EvidentlyCube**
[Relevant thread](http://forum.caravelgames.com/viewtopic.php?TopicID=39931)

### CODE REVIEW

File: FrontEndLib/ListBoxWidget.cpp [hunk 1]
```diff
@@ -660,6 +660,36 @@ bool CListBoxWidget::RemoveItem(const void* pKey)
 	return bRemoved;
 }
 
+bool CListBoxWidget::RemoveItems(const CIDSet* pKeys)
```
Comment (mrimer) [inline FrontEndLib/ListBoxWidget.cpp:663]: Suggest passing in as a const ref (i.e., "const CIDSet& keys").  Just as fast as a pointer and cleaner syntax.

**Review by mrimer — COMMENTED** [review]
Love the optimization!

Stylistic suggestion made.

**Comment by EvidentlyCube** [comment]
PR Updated!

File: FrontEndLib/ListBoxWidget.h [hunk 2]
```diff
@@ -112,6 +112,7 @@ class CListBoxWidget : public CScrollableWidget
 	virtual void   Paint(bool bUpdateRect = true);
 	bool           RemoveItem(const UINT dwKey);
 	bool           RemoveItem(const void* pKey);
+	bool           RemoveItems(const CIDSet& pKeys);
```
Comment (mrimer) [inline FrontEndLib/ListBoxWidget.h:115]: Rename: pKeys -> keys

**Review by mrimer — CHANGES_REQUESTED** [review]

**Comment by EvidentlyCube** [comment]
PR updated

**Review by mrimer — APPROVED** [review]

**Pull request merged**